pkgs/development/libraries/folks/default.nix

```diff
@@ -42,6 +43,16 @@ stdenv.mkDerivation rec {
     sha256 = "1f9b52vmwnq7s51vj26w2618dn2ph5g12ibbkbyk6fvxcgd7iryn";
   };
 
+  patches = [
+    # Fix tests with e-d-s linked with libphonenumber support
+    # https://gitlab.gnome.org/GNOME/folks/merge_requests/40
+    (fetchpatch {
+      url = "https://gitlab.gnome.org/GNOME/folks/commit/6d443480a137f6a6ff345b21bf3cb31066eefbcd.patch";
+      sha256 = "D/Y2g12TT0qrcH+iJ2umu4Hmp0EJ3Hoedh0H3aWI+HY=";
+    })
+    ./folks-eds-test-timeout.patch
```

Comment:
We should probably do https://gitlab.gnome.org/GNOME/folks/commit/9ada540e2c899b0d06d7183e54c87e7ab24a6fd3 instead.

Reply:
I tried that, and it brings the timeout to 30 instead of removing it. I think that commit was done along with other change to the test suite.

Reply:
You will need the timeout multiplier as mentioned in the commit message. https://mesonbuild.com/Release-notes-for-0-57-0.html#test-timeout-and-timeout_multiplier-value-0

But that would mean we would have to invoke `meson test` manually with the flag. Perhaps we can just ignore it for now.

Reply:
I changed to use the time out multiplier. Its much cleaner than using the patch.

Reply:
bumped the multiplier to 4, because it timed out on my server. I would like to just disable the time out, but I don't think that's a feature with the current meson version.

Reply:
Yeah, that requires Meson 0.57 (on #115435).